CONTRIBUTING.md

```diff
@@ -31,23 +33,30 @@ We recommend you to use our prebuilt Docker image to start your development work
 
 * From VSCode's command menu, run `Reopen in Container` to open your workspace in one of our pre-built Docker containers. Select the correct container config based on your local accelerator (default to `tpu-contributor` if you are not sure).
 
-* Since you are running as root in this container, change ownership of all files:
+* Since you are running as root in this container, teach `git` to recognize the repositories you
+  just cloned (outside of docker) as safe:
 
   ```bash
-  chown -R root:root .
+  git config --global --add safe.directory /workspaces/torch/pytorch
+  git config --global --add safe.directory /workspaces/torch/pytorch/xla
+  git config --global --add safe.directory /workspaces/torch/vision
   ```
 
-* Build PyTorch and PyTorch/XLA:
+* Build PyTorch, TorchVision, and PyTorch/XLA:
 
   ```bash
-  cd pytorch/
+  pushd pytorch
   # pytorch/xla requires pytorch wheel to be presented under pytorch/dist
   python setup.py bdist_wheel
   python setup.py install
-  cd xla/
+  popd
+  pushd vision
   python setup.py develop
-  # Optional: if you're using TPU, install libtpu
-  pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html
+  popd
+  cd pytorch/xla
+  pip install -e .
```

Comment:
Is that the same as `python setup.py develop`?

Reply:
Yeah, from [1] I believe they are equivalent. The `pip` method lets us install the `tpu` extras and I updated both for consistency.

[1]: https://stackoverflow.com/questions/45426794/setup-in-virtualenv-pip-install-e-vs-python-setup-py-install

Reply:
I still prefer `python setup.py develop` since it is what we used in above examples.

Reply:
SGTM